tests/clipboard/paste.ts

```diff
@@ -65,6 +77,18 @@ test('does not paste when disabled', async () => {
   )
 })
 
+test('does not paste when preventDefault is called', async () => {
```

Comment:
```suggestion
test('prevent input per paste event handler', async () => {
```
In fact it does `paste` it just doesn't manipulate the DOM.

Reply:
Oops, I knew that, not sure how I came up with that wording. Thanks!